bin/docker_build_image.sh

```diff
@@ -0,0 +1 @@
+  docker build -t "mywebsite:Dockerfile" . 
```

Comment:
minor naming change for the image and label
```suggestion
  docker build -t "al-folio:latest" . 
```

Reply:
That's right. I checked again and it is possible. I changed it accordingly.